deebotozmo/commands.py

```diff
@@ -29,6 +29,23 @@ def __init__(self, speed: str):
         })
 
 
+class SetScheduleOnce(Command):
+
+    def __init__(self, hour: str, minute: str ):
+        super().__init__("setSched", {
+            "act": "add",
+            "content": {"jsonStr":"{\"type\":\"auto\"}","name":"clean"},
+            "enable": 1,
```

Comment:
```suggestion
            "enable": int(enable),
```
So a user can disable it

Reply:
Not yet. Because for disable i think we need more. Read state back, maybe index, and some more.